lib/colord/cd-edid.c

```diff
@@ -139,6 +142,7 @@ cd_edid_convert_pnp_id_to_string (const gchar *pnp_id)
 		udev_hwdb_unref (hwdb);
 	if (udev != NULL)
 		udev_unref (udev);
+#endif
```

Comment:
Isn't there a FreeBSD specific way to do this?

Reply:
It seems that there are removed code in colord that use pnp.ids file to implement this function on non-udev systems. Should we bring it back?

Reply:
I found pnp.ids file was removed from gnome-desktop and there is no other package providing this file on FreeBSD ... We have no hwdata package, only usbids is available.